src/shellbar.scss

```diff
@@ -114,6 +165,8 @@ $block: #{$fd-namespace}-shellbar;
 
     @include fd-var-color("color", $fd-shellbar-color, --fd-color-text-5);
```

Comment:
this should be removed
Also 
@include fd-type("1");
@include fd-weight("bold");
line-height: 1.1;
These should be removed and replaced by values coming from the specs